src/Microsoft.SqlTools.ServiceLayer/DisasterRecovery/DisasterRecoveryService.cs

```diff
@@ -335,14 +334,30 @@ public void InitializeService(IProtocolEndpoint serviceHost)
                     SqlConnection sqlConn = ConnectionService.OpenSqlConnection(connInfo, "Backup");
```

Comment:
what not wrap this with a using statement?

Reply:
Because the usual case for it being closed is the task being completed - which can happen well after this request is handled. (the response is sent as soon as the task is kicked off, not when the task completes)